package/MDAnalysis/core/universe.py

```diff
@@ -1416,9 +1419,9 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
         To use a different conformer generation algorithm, like ETKDGv3:
 
         >>> u = mda.Universe.from_smiles('CCO', rdkit_kwargs=dict(
-                                         params=AllChem.ETKDGv3()))
+            ... params=AllChem.ETKDGv3()))
```

Comment:
```text
/home/runner/work/mdanalysis/mdanalysis/package/doc/sphinx/source/../../../MDAnalysis/core/universe.py.rst: WARNING: ignoring invalid doctest code: ">>> u = mda.Universe.from_smiles('CCO', rdkit_kwargs=dict(\n    ... params=AllChem.ETKDGv3()))\n>>> u.trajectory\n<RDKitReader with 10 frames of 9 atoms>"
```

Please format properly, so that this line is not invalid.